socorro/lib/libsocorrodataschema.py

```diff
@@ -270,6 +270,8 @@ def transform_schema(schema, transform_function):
     """
 
     def _transform_schema(schema, path, transform_function):
+        if "type" not in schema:
+            raise KeyError(f"'type' not in schema at {path}")
```

Comment:
Prior to this change, all you'd get was something like `KeyError("type")` and it wasn't clear where in the (very long) schema that happened. Now it shows the path which makes fixing these issues (which I keep hitting) much much easier.